docs/build-server-support/build-server/teamcity.md

```diff
@@ -17,13 +17,18 @@ TeamCity has support for meta-runners which allow custom tasks. There is a GitVe
  - [Project Link](https://github.com/JetBrains/meta-runner-power-pack/tree/master/gitversion)
 
 ## Running inside TeamCity
-When running in TeamCIty you have two options, run using **agent checkout** or use dynamic repositories.
+When running in TeamCity you have two options, run using **agent checkout** or use dynamic repositories.
 
 ### Agent checkout
 For GitVersion to pick up pull requests properly you need to promote the `%teamcity.build.vcs.branch.{configurationid}%` variable to an environment variable called `Git_Branch`
 
 Just go to your build configuration, Parameters, click Add, Name should be `env.Git_Branch`, value should be `%teamcity.build.vcs.branch.{vcsid}%` where vcsid is your VCS root id. You should get auto completion for this.
 
+For GitVersion to work with [MainLine mode](../../reference/mainline-development.md) you need to set configuration parameter `teamcity.git.fetchAllHeads = true`, because TeamCity by default fetches only current branch for building, but MainLine mode requires master for calculating version.
+More reference about this #912, #1199, #1234, #1441, #1477 and #1825
```

Comment:
These issue references won't resolve on [ReadTheDocs](https://gitversion.readthedocs.io/), so I think it's better to just reference them in the description of this PR rather than in the documentation itself.

Reply:
References removed and added to PR description